ukvm/ukvm_guest.h

```diff
@@ -264,9 +264,12 @@ struct ukvm_poll {
     int ret;
 };
 
+#define UKVM_HYPERCALL_MAX_DUMP_INFO_SIZE 100
```

Comment:
Once you make `ukvm_halt.data` a UKVM_GUEST_PTR (see below), then there is no need for this max len. Every access to UKVM_GUEST_PTR gets properly checked: no weird memory access above the unikernel memory.